MapboxSceneKit/Routes/TerrainNodeExtension+Routes.swift

```diff
@@ -0,0 +1,80 @@
+    ///   - radius: The width of the line
+    ///   - color: The color of the line
+    /// - Returns: The final PolylineNode, already added as a child of the TerrainNode
+    @discardableResult
+    @objc
+    public func addPolyline( coordinates: [CLLocation], radius: CGFloat, color: UIColor) -> PolylineNode{
+        
+        let scenePositions = coordinatesToSCNVector3(coordinates: coordinates)
+        let lineNode = PolylineNode(positions: scenePositions, radius: radius, color: color)
+        lineNode.position.y += Float(radius)
+        self.addChildNode(lineNode)
+        return lineNode
+    }
+    
+    /// Converts a set of coordinates to SCNVector3s relative to the TerrainNode, then adds a PolylineNode through those locations.
+    ///
+    /// - Parameters:
+    ///   - coordinates: Coordinates on the TerrainNode. The Polyline is drawn through each location consectutively from 0...n
+    ///   - startRadius: The width of the initial point of the line. Linearly interpolated from start to end positions.
+    ///   - endRadius: The width of the final point of the line. Linearly interpolated from start to end positions.
+    ///   - startColor: The color of the initial point of the line. Linearly interpolated through RGB color space from start to end.
+    ///   - endColor: The color of the final point of the line. Linearly interpolated through RGB color space from start to end.
+    /// - Returns: The final PolylineNode, already added as a child of the TerrainNode
+    @discardableResult
+    @objc
+    @available(iOS 10.0, *)
+    public func addPolyline(coordinates: [CLLocation], startRadius: CGFloat, endRadius: CGFloat, startColor: UIColor, endColor: UIColor) -> PolylineNode{
+        
+        let scenePositions = coordinatesToSCNVector3(coordinates: coordinates)
+        let lineNode = PolylineNode(positions: scenePositions, startRadius: startRadius, endRadius: endRadius, startColor: startColor, endColor: endColor)
+        lineNode.position.y += Float((startRadius + endRadius)/2) //average begining and ending radius for vertical offset
+        self.addChildNode(lineNode)
+        return lineNode
+    }
+    
+    @discardableResult
+    @objc
+    @available(iOS 10.0, *)
+    public func addPolyline(coordinates: [CLLocation], radii: [CGFloat], colors: [UIColor], verticalOffset: CGFloat = 0) -> PolylineNode{
+        
+        let scenePositions = coordinatesToSCNVector3(coordinates: coordinates)
+        let lineNode = PolylineNode(positions: scenePositions, radii: radii, colors: colors)
+        lineNode.position.y += Float(verticalOffset)
+        self.addChildNode(lineNode)
+        return lineNode
+        
+    }
+    
+    /// Convert coordinates to SCNVector3
+    ///
+    /// - Parameter coordinates: CLLocation coordinates to convert
+    /// - Returns: vector in terrainNode local space
+    fileprivate func coordinatesToSCNVector3(coordinates: [CLLocation]) -> [SCNVector3]
+    {
```

Comment:
spacing

Reply:
@avi-c Maybe you'll introduce swiftlint to the project? It will make everything consistent. 

You can check it out here: https://github.com/realm/SwiftLint. It's really easy to set up but takes a moment to update the errors/warnings in code.